### PR TITLE
Add BOOTSTRAP environment variable

### PR DESCRIPTION
**Pull request by Baisang**

### CODE REVIEW

File: templates/kafka_env.sh.erb [hunk 1]
```diff
@@ -6,3 +6,4 @@ fi
 
 export JAVA_HOME="<%= scope.lookupvar('kafka::server::java_home') %>"
 export ZOOKEEPER="<%= scope.lookupvar('kafka::server::zookeeper_connect') %>"
+export BOOTSTRAP=localhost:9092
```
Comment (yzernik) [inline templates/kafka_env.sh.erb]: Two minor things:
- Should it be BOOTSTRAP_SERVER
- Is the port ever going to be dynamically set to something different?

Comment (Baisang) [reply templates/kafka_env.sh.erb]: I think BOOTSTRAP is preferable, a lot easier to type. i don't think we're polluting the namespace with that (`git grep BOOTSTRAP` in puppet repo doesnt have anything) we could always go for $KAFKABOOTSTRAP but i just think we should avoid special characters and length if possible

i don't think the port will change, the default kafka port is just 9092

**Comment by yzernik** [comment]
+1

**Comment by 88manpreet** [comment]
+1 One minor thing. Can we also use the port through some already defined variable for that?

**Comment by Baisang** [comment]
@88manpreet i don't think there are any environment variables from kafka that have the port, it seems its discouraged https://kafka.apache.org/coding-guide (ctrl+f environment)

**Pull request merged**